ships/0020-shipwright-image-api.md

```diff
@@ -0,0 +1,123 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: shipwright-image-api
+authors:
+  - "@imjasonh"
+reviewers:
+  - "@shbose78"
+  - "@adambkaplan"
+  - "@SaschaSchwarze0"
+  - "@ricardomaraschini"
+approvers:
+  - "@adambkaplan"
+  - "@SaschaSchwarze0"
+creation-date: 2021-08-16
+last-updated: 2021-08-16
+status: provisional
+see-also:
+  - https://github.com/shipwright-io/community/issues/8
+---
+
+# Shipwright Image API
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions
+
+1. Is this a worthwhile and well-aligned addition to the existing Shipwright project scope?
```

Comment:
If I may: images are feedstock and the outcome of a build process, they are very much related. If we take security concerns into account it would be very useful to be able to automatically trigger multiple build processes whenever a new version for a "base" image pops up. I guess that would add towards the "worthwhile" part of the question.